cms/bundles/tests/test_viewsets.py

```diff
@@ -139,6 +141,39 @@ def test_bundle_add_view__without_permissions(self):
                 self.assertRedirects(response, "/admin/")
                 self.assertContains(response, "Sorry, you do not have permission to access this area.")
 
+    def test_date_placeholder(self):
+        """Test that the date input field displays date placeholder."""
+        response = self.client.get(self.bundle_add_url, follow=True)
+
+        datetime_placeholder = "YYYY-MM-DD HH:MM"
+        self.assertContains(
```

Comment:
Non blocking nit: I think in other places we've been using `self.assertInHTML` if we want consistency, but this also works